android/build.gradle

```diff
@@ -9,7 +9,7 @@ buildscript {
         }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.0'
+        classpath 'com.android.tools.build:gradle:3.0.1'
```

Comment:
Please note that this version of build gradle should be compatible with build tools 23.0.1